python-main.js

```diff
@@ -280,6 +281,20 @@ function translations() {
         };
     }
 
+    /* Checks URL for query parameter 'l' and translates page to that language. */
+    function setLang(message){
```

Comment:
I know this variable name follows the same pattern as a different function, but the connection between this name and the query parameters is hard to see, so probably better if we rename it to something like urlParams, or something like that.